source/blueconfig.rst

```diff
@@ -796,6 +796,13 @@ Neurodamus.
          Name of the target in start.target or user.target to receive the
          manipulation
 
+    .. blueconfig_value:: SectionConfigure
+        :type: string
+        :required: True for ConfigureAllSections
+        :unit:
+        :description:
+         A snippet of python code to perform one or more assignments involving section attributes, for all sections that have all the referenced attributes. The wildcard %s represents each section. Multiple statements are separated by semicolons. E.g., "%s.attr = value; %s.attr2 *= value2".
```

Comment:
```suggestion
         A snippet of python code to perform one or more assignments involving section attributes, for all sections that have all the referenced attributes. The wildcard %s represents each section. Multiple statements are separated by semicolons. E.g., "%s.attr = value; %s.attr2 \*= value2".
```

Is escaping of the asterisk required to avoid formatting?

Reply:
Yes, it is seems that the asterisks (in pairs) are reserved for formatting. Without the escaping here, the doc won't compile
`sonata_simulation.rst:159:Inline emphasis start-string without end-string.`

Reply:
I don't know why there was no need to excape for the blueconfig doc. I didn't have error for that.

Reply:
Ok, then let's leave it like that :)